docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -56,24 +55,24 @@ The Model Optimizer finds the last layer of the topology and removes this layer
 
   > **NOTE**: Model Optimizer can remove SoftMax layer only if the topology has one output.
 
-  > **NOTE**: For sample inference of Kaldi models, you can use the OpenVINO Speech Recognition sample application. The sample supports models with one output. If your model has several outputs, specify the desired one with the `--output` option.
+  > **NOTE**: You can use the OpenVINO Speech Recognition sample application for sample inference of Kaldi models. The sample supports models with one output. If your model has several outputs, specify the desired one with the `--output` option.
```

Comment:
```suggestion
* You can use the *OpenVINO Speech Recognition* sample application for the sample inference of Kaldi models. This sample supports models with only one output. If your model has several outputs, specify the desired one with the `--output` option.
```
This can be positioned as another CLI example. Better include a link to the sample (but I'm not sure which one is).